SWLOR.Game.Server/Feature/AbilityDefinition/Devices/IonGrenadeAbilityDefinition.cs

```diff
@@ -71,8 +71,9 @@ private void IonGrenade1()
             _builder.Create(FeatType.IonGrenade1, PerkType.IonGrenade)
                 .Name("Ion Grenade I")
                 .Level(1)
-                .HasRecastDelay(RecastGroup.IonGrenade, 30f)
-                .HasActivationDelay(1f)
+                .HasRecastDelay(RecastGroup.IonGrenade, 24f)
+                .HasRecastDelay(RecastGroup.Grenades, 6f)
```

Comment:
Same thing here - only one recast group is allowed and it should be the specific Ion Grenade group.